content/en/docs/concepts/containers/runtime-class.md

```diff
@@ -174,13 +174,10 @@ Nodes](/docs/concepts/configuration/assign-pod-node/).
 
 ### Pod Overhead
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.18" state="beta" >}}
 
 As of Kubernetes v1.16, RuntimeClass includes support for specifying overhead associated with
 running a pod, as part of the [`PodOverhead`](/docs/concepts/configuration/pod-overhead/) feature.
-To use `PodOverhead`, you must have the PodOverhead [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-enabled (it is off by default).
-
 
 Pod overhead is defined in RuntimeClass through the `Overhead` fields. Through the use of these fields,
```

Comment:
optional fix:
```suggestion
Pod overhead is defined in RuntimeClass through the `overhead` fields. Through the use of these fields,
```

Reply:
The top level field is Overhead, so I think it is appropriate as is?